### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by munjalSaksham71**
In this PR: I have been fixing Bugs:

1. Getting data Realtime, Now we don't have to reload to get data
2. making keep it logo to a link
3. Sidebar icons should be of white-color

NOTE: still ```color change```  is pending as i have to make edit functionality first- this will be done in next PR.

### CODE REVIEW

File: src/components/ViewNote/ViewNote.css [hunk 1]
```diff
@@ -12,7 +12,7 @@
 }
 
 .card_icons {
-    font-size: large;
+    font-size: 24px;
```
Comment (Raghav888) [inline src/components/ViewNote/ViewNote.css:15]: Use rem

**Review by Raghav888 — APPROVED** [review]
Good structure

File: src/components/ViewNote/ViewNote.css (hunk 1 above)
Comment (munjalSaksham71) [reply src/components/ViewNote/ViewNote.css:15]: Hey raghav, i have been using px as i don't want it to change irrespective of responsiveness. I want that icon to be exact 24px at every possible width

**Review by JantuDeb — APPROVED** [review]
Overall the code looks find. 
I observed some issues with create note, previous notes get remove from the screen for few seconds when I add a new note.

**Comment by munjalSaksham71** [comment]
> Overall the code looks find. I observed some issues with create note, previous notes get remove from the screen for few seconds when I add a new note.

Fixed Now, As useEffect dependency array is constantly changing and the component is rerendering infinitely.

**Pull request merged**